doc/lsst.meas.extensions.multiprofit/index.rst

```diff
@@ -6,23 +6,37 @@
 lsst.meas.extensions.multiprofit
 ################################
 
-.. Tasks for running the MultiProFit astronomical source modelling code on
-.. multiband exposures/catalogs.
-
-.. .. _lsst.meas.extensions.multiprofit-using:
-
-.. Using lsst.meas.extensions.multiprofit
-.. ======================================
-
-``lsst.meas.extensions.multiprofit`` implements separate tasks for PSF and source model fitting
-using ``lsst.multiprofit``. The tasks depend on more generic ``lsst.pipe.tasks.PipelineTask``
-classes defined in ``lsst.pipe.tasks.fit_coadd_psf`` and ``lsst.pipe.tasks.fit_coadd_multiband``.
-Note that as implied by the names, the PSF fitting task is single-band only.
-
-Because these tasks produce patch-based Arrow/Astropy tables, a generic task to consolidate
-the results into a single multiband tract-based table is also provided. It is anticipated that
-this functionality will be folded into the tasks producing objectTable_tract instances in
-``lsst.pipe.tasks.postprocess``.
+:py:mod:`lsst.meas.extensions.multiprofit` implements separate tasks for PSF and source model fitting
+using :py:mod:`lsst.multiprofit`. The tasks depend on more generic :py:class:`lsst.pipe.tasks.PipelineTask`
+classes defined in :py:mod:`lsst.pipe.tasks.fit_coadd_psf` and :py:mod:`lsst.pipe.tasks.fit_coadd_multiband`.
```

Comment:
The PSF fitting is single-band only and fits the coadd PSF image. I suppose `fit_coadd_multiband` could have been called `fit_coadd_objects_multiband` but that is arguably redundant since objects are generated only from coadds.